plugin-taskbar/lxqttaskbar.cpp

```diff
@@ -54,6 +56,7 @@ using namespace LXQt;
 ************************************************/
 LXQtTaskBar::LXQtTaskBar(ILXQtPanelPlugin *plugin, QWidget *parent) :
     QFrame(parent),
+    m_pSignalMapper(new QSignalMapper(this)),
```

Comment:
Please, try to be coherent with the naming convetion used in the scope of the file/class